schemas/com.snowplowanalytics.snowplow/media_player_session/jsonschema/1-0-0

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity for media player events that tracks a session of a single media player usage",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_session",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "sessionId": {
+      "type": "string",
+      "format": "uuid",
+      "description": "An identifier for the session."
+    },
+    "startedAt": {
+      "type": [
+        "null",
+        "string"
+      ],
+      "description": "Date-time timestamp of when the session started.",
+      "format": "date-time"
+    },
+    "pingIterval": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Interval (seconds) in which the ping events will be sent. Default (specified in the tracker media docs) is assumed if not specified.",
+      "minimum": 0,
+      "maximum": 65535
+    },
+    "timeSpent": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Total seconds user spent on this session.",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "timePlayedContent": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Total seconds user spent playing content (excluding ads).",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "contentDurationPlayed": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Total seconds of the content played. Each part of the content played is counted once (i.e., counts rewinding or rewatching the same content only once). Playback rate does not affect this value.",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "timeSpentAds": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Total seconds that ads played during the session.",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "totalAds": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Number of ads played.",
+      "minimum": 0,
+      "maximum": 65535
```

Comment:
Do we want to add the number of ad breaks as well? Also, integer?

Reply:
I think that makes sense, added `totalAdBreaks` and fixed integer type.